lib/logger.js

```diff
@@ -403,7 +403,7 @@ class Logger extends EventEmitter {
     , agent
     , headers: {
         'Content-Type': 'application/json; charset=UTF-8'
-      , 'Authorization': 'Basic ' + Buffer.from(`${key}:`).toString('base64')
+      , 'apikey': `${key}`
```

Comment:
Are we absolutely SURE this isn't a breaking change?!  There's no harm doing a `major` if we're not sure.

Reply:
I think it is as this is a major change, because it changes how clients handle authentication - which is a security concern. And we're not giving people an option to choose. If for some reason they are more comfortable using basic for the masking capabilities, they would have no choice but to pin to a specific patch version.

Is there a reason we don't make it configurable?

Reply:
Yes, I think it should be major.  And I agree with all of @esatterwhite 's points.  It feels strange to force people to use apikey.  Would all of our users have one set up?

Reply:
> I think it is as this is a major change, because it changes how clients handle authentication - which is a security concern. 

I can make it configurable.  What should be the default @esatterwhite, @darinspivey?

Reply:
It feels extremely clunky to me to push this kind of nuance on to clients + customers when we are fully in control of this.

We should support the basic authentication scheme at both points of ingestion. That is an HTTP standard. We should support that.

The apikey thing is old baggage that We should deprecate and push people to use the standard.